tasking/pulp/tasking/services/worker_watcher.py

```diff
@@ -100,3 +102,43 @@ def handle_worker_offline(event):
     msg = _("Worker '%(worker_name)s' shutdown") % event_info
     _logger.info(msg)
     delete_worker(event_info['worker_name'], normal_shutdown=True)
+
+
+def delete_worker(name, normal_shutdown=False):
+    """
+    Delete the :class:`~pulp.app.models.Worker` with name from the database, cancel any associated
+    tasks and reservations
+
+    If the worker shutdown normally, no message is logged, otherwise an error level message is
```

Comment:
I think the comma after logged should either be a semicolon or split into two separate sentences.